phpfpm/service.sls

```diff
@@ -8,7 +8,7 @@
 
 extend:
 # Vanilla Ubuntu 14.04 packages
-{% if php_versions|length == 0 %}
+{% if php_versions|length == 0 and webserver_edition == 'vanilla' %}
```

Comment:
@alexander2701 This change prevents from installing "other" PHP versions in conjunction with Zend Server. Is this something we want?

Reply:
php-fpm is added in install.sls, but only if php_versions|length == 0 and webserver_edition == 'vanilla'. "install php-fpm" and  "extend php-fpm" should have the same conditions, otherwise it may be extended without being installed (=failure)

Reply:
Yeah, but the top check is for stock PHP-FPM / the one used by Zend. If we merge this, that will likely break?